frontend/testing/playwright/pages/OrgLibraryPage/CodeLists.ts

```diff
@@ -90,4 +92,61 @@ export class CodeLists extends BasePage {
 
     await expect(codeList).toBeVisible();
   }
+
+  public async clickOnCodeListAccordion(title: string): Promise<void> {
+    await this.page
+      .getByTitle(
+        this.textMock('app_content_library.code_lists.code_list_accordion_title', {
+          codeListTitle: title,
+        }),
+      )
+      .click();
+  }
+
+  public async verifyNumberOfRowsInTheCodelist(numberOfRows: number): Promise<void> {
+    for (let i = 1; i <= numberOfRows; i++) {
+      const valueRow = this.page.getByRole('textbox', {
+        name: this.textMock('code_list_editor.value_item', { number: i.toString() }),
+        exact: true,
+      });
+
+      await expect(valueRow).toBeVisible();
+    }
+  }
+
+  public async typeInSearchBox(searchTerm: string): Promise<void> {
+    await this.page
+      .getByRole('searchbox', {
+        name: this.textMock('app_content_library.code_lists.search_label'),
+      })
+      .fill(searchTerm);
+  }
+
+  public async clickOnDeleteCodelistButton(): Promise<void> {
+    await this.page
+      .getByRole('button', {
+        name: this.textMock('app_content_library.code_lists.code_list_delete'),
+      })
+      .click();
+  }
+
+  public async listeToAndWaitForConfirmDeleteCodeList(codeListTitle: string): Promise<void> {
```

Comment:
Typo in function name
```suggestion
  public async listenToAndWaitForConfirmDeleteCodeList(codeListTitle: string): Promise<void> {
```